_studio/shared/include/mfxvideo++int.h

```diff
@@ -55,7 +55,7 @@ typedef struct {
     unsigned char  Data4[8];
 } GUID;
 
-static int operator==(const GUID & guidOne, const GUID & guidOther)
+static inline int operator==(const GUID & guidOne, const GUID & guidOther)
```

Comment:
According to compiler warning: 'static' function 'operator==' declared in header file should be declared 'static inline' [-Wunneeded-internal-declaration]

Reply:
ok